config.go

```diff
@@ -170,7 +170,8 @@ type config struct {
 	SigNet               bool          `long:"signet" description:"Use the signet test network"`
 	SigNetChallenge      string        `long:"signetchallenge" description:"Connect to a custom signet network defined by this challenge instead of using the global default signet test network -- Can be specified multiple times"`
 	SigNetSeedNode       []string      `long:"signetseednode" description:"Specify a seed node for the signet network instead of using the global default signet network seed nodes"`
-	TestNet3             bool          `long:"testnet" description:"Use the test network"`
+	TestNet3             bool          `long:"testnet" description:"Use the test network (version 3)"`
+	TestNet4             bool          `long:"testnet4" description:"Use the test network (version 4)"`
```

Comment:
We should add this to the sample btcd conf: https://github.com/btcsuite/btcd/blob/master/sample-btcd.conf